.circleci/config.yml

```diff
@@ -173,7 +173,14 @@ jobs:
 
       - store_artifacts:
           path: build/artifacts/armadillo-compose.zip
+          destination: armadillo-compose.zip
```

Comment:
include version in file name?

Reply:
Now we have a first stab @ v4.0.0 I think and will test we do not have to version the CircleCI artifacts but need to transform nyx release by *adding* the version.